### ISSUE TITLE
Broken link in documentation

### ISSUE THREAD

**Issue by joshpurvis** [hectorj/terraform-provider-googlesiteverification#15]
Hi, thanks for making this! 

I almost have it working, but running into some credential issues. This internal link that you mention in the readme no longer exists, so I don't know which section it's pointing to: https://www.terraform.io/docs/providers/google/guides/provider_reference.html#credentials-1

Could you let me what I should be looking at?

I have the google provider working using the credentials argument.

```
provider "google" {
  project     = var.project_name
  region      = var.region
  credentials = var.credentials_path
}
```

Here's the error i'm getting:

```Error: Post "https://www.googleapis.com/siteVerification/v1/token?alt=json&prettyPrint=false": oauth2: cannot fetch token: 400 Bad Request
Response: {"error":"invalid_scope","error_description":"Invalid OAuth scope or ID token audience provided."}

  on dns.tf line 13, in data "googlesiteverification_dns_token" "default":
  13: data "googlesiteverification_dns_token" "default" {
```

I should also note the install command you have no longer works for > 0.13 terraform, had to manually create the directory structure per terraform's upgrade guide and using `required_providers` block

**Comment by hectorj** [comment]
Hi, and thank you for the reports!

For the doc it's now : https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#authentication

- If you are logged in via `gcloud auth application-default login` it should work.
- If you are running on GCP with a service account it should work.
- The `GOOGLE_APPLICATION_CREDENTIALS` env var should work.

For the installer, I'll drop it and go through https://www.terraform.io/docs/registry/providers/publishing.html

**Comment by hectorj** [comment]
I'll have to work on the documentation but here it is: https://registry.terraform.io/providers/hectorj/googlesiteverification

**Comment by hectorj** [comment]
Fixed by 9076bce, 2bec946, and b54cdc7

**Comment by joshpurvis** [comment]
I don't know what my issue was before, as I was using the environment variable, but after installing via the registry it all worked. Thank you!
